.github/workflows/release_crates.yml

```diff
@@ -61,6 +60,8 @@ jobs:
         run: |
           git checkout master
           cargo login ${{ secrets.CARGO_REGISTRY_TOKEN }}
-          cargo release --no-confirm --execute --sign ${{ steps.crate_version.outputs.version }}
+          cargo publish --no-confirm --execute
```

Comment:
Why do we not pss the version anymore? :)

Reply:
`publish` doesn't have a version argument, it publishes all packages which are not published yet.

I think the problem with the current setup is that `cargo release` creates a new commit, and my hope is that with this setup that is skipped. The new commit cannot be pushed due to the branch protection rules.

Reply:
Yup, I understand the second part, I was just a bit suspicious if it can determine the version properly. In theory Cargo.toml works but Not sure.